doc/source/whatsnew/v0.21.0.txt

```diff
@@ -129,8 +129,33 @@ e.g., when converting string data to a ``Categorical`` (:issue:`14711`,
    dtype = CategoricalDtype(categories=['a', 'b', 'c', 'd'], ordered=True)
    s.astype(dtype)
 
+One place that deserves special mention is in :meth:`read_csv`. Previously, with
+``dtype={'col': 'category'}``, the returned values and categories would always
+be strings.
+
+.. ipython:: python
+
+   from pandas.compat import StringIO
```

Comment:
in general we put this in the hidden code block at the top of the file, as people shouldn't use this from pandas, but just import it themselves